### DISCUSSION TITLE
Morgen + ? (iOS)

### DISCUSSION THREAD

**Discussion by KjellConnelly** [BryanJenksCommunity/FAQ#132] [Q&A] [answered]
I'm trying to get into Morgen due to your video and finding it helpful. And in the process, since I use my phone more regularly than my laptop (though I do use it in chunks), I'm finding that I need an app for calendars on my iPhone. Any suggestions?

I'm currently just using my iCloud calendars and using the Apple Calendar app. It does a decent job. Just wondering if you have any suggestions coming from one adhd guy to another who wants streamlined experiences without being overwhelmed by shininess and fluff?

I'm planning to syncing both apps with all my iCloud calendars (some of which are shared with my wife to help us plan our time together/away/with kids etc)

**Comment by tallguyjenks** [comment, marked as answer]
at the moment morgen doesnt have an iOS app sadly, so i use ical on my phone and just have all my calendars sync to morgen so i use morgen on my laptop not my phone. hopefully they come out with some of those features soon!
